open-hackathon-server/src/hackathon/user/user_manager.py

```diff
@@ -82,6 +83,34 @@ def login(self, provider, context):
         else:
             return self.__oauth_login(provider, context)
 
+    def update_user_operation_time(self):
+        """Update the user's last operation time.
+
+        :rtype:bool
+        :return True if success in updating, return False if token not found or token is overtime.
+        """
+        if HTTP_HEADER.TOKEN not in request.headers:
+            return False
+
+        user = self.__validate_token(request.headers[HTTP_HEADER.TOKEN])
+        if user is None:
+            return False
+
+        users_operation_time[user.id] = self.util.get_now()
+        return True
+
+    def check_user_online_status(self):
+        """Check whether the user is offline. If the answer is yes, update its status in DB."""
+        overtime_user_ids = []
+        for user_id in users_operation_time:
```

Comment:
A good convention in python: use `map` whenever you can.  Search if wonder why.  Just a best practice so you can leave as it is